pcap-int.h

```diff
@@ -293,7 +288,6 @@ struct pcap {
 	 */
 	int bpf_codegen_flags;
 
-#if !defined(_WIN32) && !defined(MSDOS)
```

Comment:
This should be either `#ifndef _WIN32` or `#if !defined(_WIN32)`, as Windows doesn't have UN\*X-style file descriptors on which you can do stuff such as `select()`, except for sockets, and libpcap doesn't offer a "get selectable descriptor" call on Windows for that reason.

Reply:
like; e80574e2 ?

Reply:
Yes.